tests/installation/logs_from_installation_system.pm

```diff
@@ -36,6 +36,15 @@ sub run {
     my $dasd_path = get_var('DASD_PATH', '0.0.0150');
     select_console 'install-shell';
 
+    # permit ssh root login as it is disabled in "Common Criteria" "System Role" system
+    if (check_var('SYSTEM_ROLE', 'Common_Criteria') && is_sle) {
+        assert_script_run('mount /dev/vda2 /mnt');
```

Comment:
Hi Lili,
I think it is NOT the best way to add permit root login here if we have to remount root file system here?
For arm/x86_64/s390zkvm VM, root disk should be /dev/vda2 if we set the device driver to "virtio-blk", but how about zvm? how about UEFI mode? how about BM with ipmi controll?

We hardcode disk name vda here, it might introduce new issue.

Reply:
Then  do you have a good way to handle this?  I have no idea atm.